docs/concepts/05_credentials/02_ctypes.md

```diff
@@ -2,124 +2,131 @@
 import ctype from '@site/scripts/out/ctype.json.raw!=!raw-loader!@site/scripts/out/ctype.json';
 
-CTypes are data types specific to KILT that define the structure of a claim (i.e., its data model).
-CTypes are based on JSON Schema, a standard used to annotate and validate JSON documents.
+Claim types (CTypes) are data types specific to KILT that define the structure of a claim (i.e., its data model).
+CTypes are based on [JSON Schema](https://json-schema.org/), a standard used to annotate and validate JSON documents.
 The schema defines which properties exist and what their type should be, e.g., a string, a number, an object, etc.
 
-## JSON Schema
+## CType model JSON schema
 
-KILT uses [JSON Schema](https://json-schema.org/) (currently draft-07) to validate and annotate data in a strict format.
-This data format is used to define [CType models](https://github.com/KILTprotocol/sdk-js/blob/master/packages/core/src/ctype/CType.schemas.ts).
-The following are all required properties of the schema:
+The following are all required properties of the JSON schema for [CType models](https://github.com/KILTprotocol/sdk-js/blob/master/packages/core/src/ctype/CType.schemas.ts):
 
-- **Identifier**: `$id` in the format `kilt:ctype:0x{cTypeHash}`.
-- **Reference to CType metaschema (`$schema`)**: Describes what a valid CType must looks like. The latest metaschema is accessible at [ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/](ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/).
-- **Title**: Defines a user-friendly name for the CType that makes it easier for users to contextualize.
-- **Properties**: Set of fields (e.g., name, birthdate) that the CType can contain, and hence that the Claimer can have attested.
-- **Type**: Is always `"object"`,  instructing the JSON schema validator to expect an object (where each property is a claim about the Claimer in the credential).
-- **Additional properties**: In newer CTypes, *additionalProperties* must be present and must be set to `false`, restricting allowable claims in a credential to those listed in `properties`.
+-   `$id`: An **identifier**: in the format `kilt:ctype:0x{cTypeHash}`.
+-   `$schema`: A **reference to CType metaschema**: Describes what a valid CType must looks like. You can find the latest metaschema on IPFS at the following address [ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/](ipfs://bafybeiah66wbkhqbqn7idkostj2iqyan2tstc4tpqt65udlhimd7hcxjyq/).
+-   `title`: A user-friendly name for the CType that makes it easier for users to contextualize.
+-   `properties`: A set of fields (e.g., name, birth date) that the CType can contain, and that the Claimer can have attested. [Read more details about properties below](#properties).
+-   `type`: An object containing properties for a claim about the Claimer in the credential.
+<!-- TODO: Since when? -->
```

Comment:
Since which version?

Reply:
So should it be draft-2 or CType Metaschema (V1)? or does it not really matter?

Reply:
I will say V1